src/loader.py

```diff
@@ -47,14 +46,14 @@ def load_documents(path: str):
         recursive=True,
         required_exts=[
             ".pdf",
-            # ".docx",
-            # ".py",
+            ".docx",
+            ".py",
             ".txt",
-            # ".md",
+            ".md",
             ".png",
             ".jpg",
             ".jpeg",
-            # ".ts",
+            ".ts",
```

Comment:
If you're planning on un-commenting file support, could you also just add some test files?

Reply:
Sure! I have created a [new pull request](https://github.com/iyaja/llama-fs/pull/56) for this